api/app/Mail/BeforeEventMailForOwner.php

```diff
@@ -45,6 +46,8 @@ public function build()
             ->line(Lang::get('Event Date/Time in UTC timezone:'))
             ->line($this->event->start_date)
             ->line(Lang::get("Event Date/Time in {$this->event->timezone} timezone:"))
-            ->line($eventTimeInUTCZone->timezone($this->event->timezone));
+            ->line($eventTimeInUTCZone->timezone($this->event->timezone))
+            ->line(Lang::get('Meeting link'))
+            ->line(new HtmlString('<a href="' . $this->event->zoom_meeting_link . '">' . "{$this->event->zoom_meeting_link}" . '</a>'));
```

Comment:
The meeting link must be added only in case it EventType has `zoom` location type